### PR TITLE
✨Python ingest support for bridge server

### PR DESCRIPTION
**Pull request by DanielVF**
This code adds some events for the bridge server to listen to, and the start of python API for working with the origin blockchain data.

It is expected we'll remove the python code from this repo once it's up and going on the bridge server.

### CODE REVIEW

**Comment by joshfraser** [comment]
Agreed, let's get this started on the bridge server. @tyleryasaka you probably have more context on how that is setup if you want to tackle porting it over.

**Comment by tyleryasaka** [comment]
@Gzing This is the part of the bridge server you're working on I believe. I'll let you integrate this with the work you're doing as needed.

File: packages/originprotocol-python/originprotocol/origin.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import json
+import time
+from web3 import Web3
+from web3.contract import Contract
+
+
+class Origin:
+    def __init__(self, rpc_server, ipfs_server="https://gateway.originprotocol.com/"):
```
Comment (amites) [inline packages/originprotocol-python/originprotocol/origin.py]: `str` as default value is mutable, better to use `None` and set in method

**Comment by DanielVF** [comment]
@amites Thanks for the suggested changes! This is useful.

It looks like Gagan has already pulled the Python API portion of this PR into the bridge_server repo, so I'm closing this PR, and moving just the needed contract events into another PR.

**Pull request closed without merging**